src/pages/standup.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import styles from '@/styles/standup.module.scss';
+
+const StandUp: FC = () => {
+    return (
+        <>
+            <Layout>
+                <Head title="Standup" />
+                <div className="container">
+                    <div className={styles.standupContainer}>
+                        <div className={styles.standupBanner}>
+                            <p>
+                                You have <span>2 missed</span> Standup updates
+                                this week
+                            </p>
+                            <p>Let&apos;s try to avoid having zero days </p>
+                            <div className={styles.buttonContainer}>
+                                <button className={styles.continueButton}>
+                                    continue
+                                </button>
+                                <button className={styles.updateButton}>
+                                    Fill Old Updates
+                                </button>
+                            </div>
+                        </div>
+                        <div className={styles.standupUpdateContainer}>
+                            <h1>Standup Update</h1>
+                            <div className={styles.standupForm}>
+                                <div className={styles.yesterdayUpdate}>
+                                    <h1 className={styles.updateHeading}>
+                                        On March 11, 2023
```

Comment:
if I am filling the standup update on 29 October 2023 should I still be showing 11th march?

Reply:
ritik, API needs to be integrated.

Reply:
What API needs to be integrated to show the current date? Doesn't your browser have the capability to show the current date?

If my backend is in the US, I will send you the US date and then you will be filling standup for 10 March on 11 March in IST?